doc/classes/Control.xml

```diff
@@ -164,11 +164,12 @@
 			<return type="Object" />
 			<param index="0" name="for_text" type="String" />
 			<description>
-				Virtual method to be implemented by the user. Returns a [Control] node that should be used as a tooltip instead of the default one. The [param for_text] includes the contents of the [member tooltip_text] property.
+				Virtual method to be implemented by the user. Returns a [Control] node that should be used as a tooltip instead of the default one. The [param for_text] includes the return value of [method get_tooltip].
```

Comment:
It's not "includes" anymore, I assume. It just "is" what `get_tooltip` would return(?). Very unsure.
```suggestion
				Virtual method to be implemented by the user. Returns a [Control] node that should be used as a tooltip instead of the default one. [param for_text] is the returned string from [method get_tooltip].
```

Reply:
The behavior has not changed. `for_text` is always the return value of `get_tooltip()`.